kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/write/row/BigQueryWriter.java

```diff
@@ -154,6 +156,10 @@ public void writeRows(PartitionedTableId table,
           // rate limit exceeded error
           logger.warn("Rate limit exceeded for table {}, attempting retry", table);
           retryCount++;
+        } else if (err.getCode() == UNKNOWN_CODE && err.getCause() != null && err.getCause() instanceof IOException){
+          //Retry when IO exceptions occur
```

Comment:
Nit: it'd be nice to keep the style for these comments consistent
```suggestion
          // IO error
```